src/sentry/utils/snuba.py

```diff
@@ -1237,6 +1245,7 @@ def _passthrough_arg(arg):
             for i, arg in enumerate(func_args):
                 if i == 0:
                     if isinstance(arg, (list, tuple)):
+                        assert isinstance(arg, list)
```

Comment:
hmm yeah this points to unsoundness -- a safer change would be to remove the check for tuple I think?